index.html

```diff
@@ -148,14 +148,14 @@ <h3>Manifest Contexts</h3>
 				</section>
 
 				<section id="audio-type">
-					<h3>Publication Type</h3>
+					<h3>Publication Conformance</h3>
 
-					<p>An audiobook's< manifest MUST define its <dfn>Publication Type</dfn> using the <code data-dfn-for="PublicationManifest"><dfn>type</dfn></code> term&#160;[[!pub-manifest]], [[!json-ld]]. The type MUST be <a href="https://schema.org/Audiobook"><code>Audiobook</code></a>&#160;[[schema.org]].</p>
+					<p>An audiobook's< manifest MUST define its <dfn>Conformance URL</dfn> using the <code data-dfn-for="PublicationManifest"><dfn>conformsTo</dfn></code> term&#160;[[!pub-manifest]], [[!json-ld]]. The conformance URL MUST be <a href="https://www.w3.org/TR/audiobooks"><code>https://www.w3.org/TR/audiobooks</code></a>&#160;[[schema.org]].</p>
 
 					<pre class="example" title="Setting a publication's type to Audiobook.">
```

Comment:
> Conformance URL MUST be

should say

> Conformance URL MUST include

(The value of `conformsTo` is an array of URLs which may include, e.g., a11y conformance references, too.